### PR TITLE
Add Avid Link.app v2020.6

### PR DESCRIPTION
**Pull request by targendaz2**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by targendaz2** [comment]
I could not find a suitable page to use as an appcast.

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/86934#discussion_r463959354

File: Casks/avid-link.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  url "http://cdn.avid.com/AvidLink/#{version}/B0C4DA46/Avid_Link_#{version[2..]}_Mac.dmg"
+  appcast "https://avid.secure.force.com/pkb/articles/en_US/User_Guide/Avid-Link-Documentation"
+  name "Avid Link"
+  homepage "https://www.avid.com/products/avid-link"
+
+  auto_updates true
+
+  pkg "AvidLink_Installer.pkg"
+
+  uninstall_script = "#{appdir}/Avid_Uninstallers/Avid Link/Avid Link Uninstaller.app/Contents/MacOS/AvidUninstaller"
```
Comment (vitorgalvao) [inline Casks/avid-link.rb:14]: No point in a variable that will only be used once. Use it directly further down.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by targendaz2** [comment]
Closing this. App will be installed as part of cask `sibelius` ( #86934 )

**Pull request closed without merging**